doc/api_ref/tls.rst

```diff
@@ -601,7 +604,7 @@ The full code for a TLS server using asio is in `src/cli/tls_proxy.cpp`.
     class Server_Credentials : public Botan::Credentials_Manager
     {
        public:
-	  Server_Credentials() : m_key(Botan::PKCS8::load_key("botan.randombit.net.key"))
+          Server_Credentials() : m_key(Botan::PKCS8::load_key(Botan::DataSource_Stream("botan.randombit.net.key")))
```

Comment:
In VS this still results in a warning:
```
warning C4239: nonstandard extension used: 'argument': conversion from 'Botan::DataSource_Stream' to 'Botan::DataSource &'
```

So may still not work on GCC/Clang. But did not see an easy fix without any bigger code changes. Any ideas?

Reply:
Found a way to avoid the warning.